src/site/includes/header.html

```diff
@@ -26,11 +26,7 @@
   {% endif %}
 
   <!-- Add metatags. -->
-  {% if drupalTags %}
-    {% include "src/site/includes/metatags.drupal.liquid" %}
-  {% else %}
-    {% include "src/site/includes/metatags.liquid" %}
-  {% endif %}
+  {% include "src/site/includes/metatags.drupal.liquid" with drupalTags = drupalTags %}
```

Comment:
```suggestion
  {% include "src/site/includes/metatags.drupal.liquid" %}
```

If `drupalTags` is already a variable in scope, then we don't need to explicitly pass it to the metatags template since all child templates have access to all variables from parent templates 🙂